test/test_service.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# coding=utf-8
+
+import json
+
+
+def test_download_mnist(mnist_class):
+    assert mnist_class.image_count == 10000
+
+
+def test_download_mnist_xshape(mnist_class):
+    assert mnist_class.x_test.shape == (10000, 28, 28)
+
+
+def test_download_mnist_yshape(mnist_class):
+    assert mnist_class.y_test.shape == (10000,)
+
+
+def test_model_loaded(mnist_class):
```

Comment:
Might be helpful to check that the ONNX model is well-formed? 

```
def test_model_form(model):
    onnx.checker.check_model(model)
```

Reply:
This is done when the MNIST class is instantiated